### PR TITLE
Expose class flags for debug and display

### PR DESCRIPTION
**Pull request by Roms1383**
A minor change to wrap `red::CClass_Flags` in a `ClassFlags` for display and debug purpose (with `Class::flags`).

### CODE REVIEW

File: src/types/rtti.rs [hunk 1]
```diff
@@ -262,6 +267,34 @@ impl Drop for Class {
...
+            format_args!("always transient: {}", self.0.isAlwaysTransient()),
+            format_args!("import only: {}", self.0.isImportOnly()),
+            format_args!("private: {}", self.0.isPrivate()),
+            format_args!("protected: {}", self.0.isProtected()),
+            format_args!("test only: {}", self.0.isTestOnly()),
+            format_args!("savable: {}", self.0.isSavable()),
+            format_args!("b10: {}", self.0.b10()),
+        )
+    }
+}
```
Comment (jac3km4) [inline src/types/rtti.rs:296]: I'd probably rather implement fmt::Debug and format it like a struct of booleans

Comment (Roms1383) [reply src/types/rtti.rs]: refactored in 86e7e5a

Comment (Roms1383) [reply src/types/rtti.rs]: @jac3km4 that being said the fields are still formatted as previously, I can change them from e.g. `scripted class` to `isScriptedClass`.
the current debug formatting also "hides" the inner struct `red::CClass_Flags`, not sure if it should appears in the logs.
waiting for your review before marking it as resolved.

File: src/types/rtti.rs [hunk 2]
```diff
@@ -262,6 +267,31 @@ impl Drop for Class {
...
 
+#[repr(transparent)]
+pub struct ClassFlags(red::CClass_Flags);
+
+impl fmt::Debug for ClassFlags {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("ClassFlags")
+            .field("abstract", &self.0.isAbstract())
+            .field("native", &self.0.isNative())
+            .field("scripted class", &self.0.isScriptedClass())
```
Comment (jac3km4) [inline src/types/rtti.rs:278]:
```suggestion
            .field("is_scripted_class", &self.0.isScriptedClass())
```
I'd try to make the names be valid identifiers (no spaces), also I generally prefer to prefix booleans with `is_`, `has_` etc

Comment (Roms1383) [reply src/types/rtti.rs]: done in 439db76

File: src/types/rtti.rs (hunk 1 above)
Comment (Roms1383) [reply src/types/rtti.rs]: completed in 439db76

**Pull request merged**